cumulusci/tasks/bulkdata/tests/utils.py

```diff
@@ -7,13 +7,18 @@
 )
 from cumulusci.tests import util as cci_test_utils
 
+CURRENT_SF_API_VERSION = "48.0"
```

Comment:
@prescod should we update it to the actual current version, 54.0? Or do that separately?